Source/WebCore/platform/text/LocaleICU.cpp

```diff
@@ -177,12 +174,12 @@ static String getDateFormatPattern(const UDateFormat* dateFormat)
         return emptyString();
 
     UErrorCode status = U_ZERO_ERROR;
-    int32_t length = udat_toPattern(dateFormat, TRUE, 0, 0, &status);
+    int32_t length = udat_toPattern(dateFormat, true, 0, 0, &status);
```

Comment:
Oh, I think using `TRUE` is right here because this is of `UBool` type.

Reply:
The TRUE/FALSE macros in icu are apparently no longer meant to be defined unless you explicitly ask for it if I read https://github.com/unicode-org/icu/blob/2c5e021f6d33bb5d3c091a4abf61ab5ccf15f93b/icu4c/source/common/unicode/umachine.h#L235-L285 correctly at least in current versions.